packages/storage-mixin/lib/backends/disk.js

```diff
@@ -21,7 +21,7 @@ if (_.isEmpty(fs)) {
    */
   try {
     /* eslint no-undef: 0 */
-    fs = window.require('electron').remote.require('fs');
+    fs = window.require('@electron/remote').require('fs');
```

Comment:
Ahh, okay, now I see why it's in the depcheck. Hmm, I think this should be safe to replace with just require instead of window.require. Maybe worth trying?

Reply:
Also, looking even close on this, I think we don't even need to use electron remote here 🤔

Reply:
Checking what happens if I just remove that.